Servers/controllers/vendorRisk.ctrl.ts

```
@@ -1,16 +1,27 @@
+import { sequelize } from "../database/db";
 import { STATUS_CODE } from "../utils/statusCode.utils";
 import {
   createNewVendorRiskQuery,
   deleteVendorRiskByIdQuery,
+  getAllVendorRisksAllProjectsQuery,
   getVendorRiskByIdQuery,
   getVendorRisksByProjectIdQuery,
   updateVendorRiskByIdQuery,
 } from "../utils/vendorRisk.util";
 import { VendorRisk } from "../models/vendorRisk.model";
 import { sequelize } from "../database/db";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'sequelize'. Consider to delete it or rename it.

'sequelize' is defined here:


(lint/suspicious/noRedeclare)